virttest/libvirt_xml/vm_xml.py

```diff
@@ -2611,3 +2613,35 @@ def marshal_to_iothreads(tag, attr_dict, index, libvirtxml):
         if tag != 'iothread':
             return None
         return attr_dict['id']
+
+
+class VMFEATURESHPTXML(base.LibvirtXMLBase):
+
+    """
+    Hpt tag XML class of features tag
+
+    Element:
+        resizing:               text
+        maxpagesize_unit:       attribute
+        maxpagesize:            int
```

Comment:
suggest add example xml here to make it more understandable

Reply:
Updated although I don't see any other xml class has examples of their own in this file.